src/components/StatusDot/StatusDot.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { getRadii, pxToRem } from '../../utils';
+
+const StatusDotColors = {
+  good: '100, 244, 81',
+  bad: '242, 46, 67',
+  neutral: '84, 129, 217',
+  inactive: '216, 216, 216',
+};
+
+const StatusDotComponent = Styled.div`
+  width: ${pxToRem(12)};
+  height: ${pxToRem(12)};
+  border-radius: ${getRadii('round')};
+  background-color: rgba(${({ color }) => StatusDotColors[color]});
+  border: 2px solid white;
+  box-shadow: 0 0 12px 0px rgba(${({ color }) =>
+    `${StatusDotColors[color]}, 0.5`});
+`;
+
+const StatusDot = ({ color }) => (
```

Comment:
Missing type annotation.

```suggestion
const StatusDot = ({ color }: {color: keyof typeof StatusDotColors}) => (
```